deno/send_message/README.md

```diff
@@ -0,0 +1,101 @@
+# SendMessage() 
+
+A sample Deno Cloud Function sending a message using a specific channel to a receiver
+
+Supported channels are `SMS`, `EMAIL` ,`DISCORD` and `TWITTER`.
+
+
+_Example function payloads:_
+
+```json
+// Sending SMS
```

Comment:
It would also be better to remove the comments and separate each example using italicized text instead, as [other](https://github.com/open-runtimes/examples/tree/main/java/short_url_generator) [examples](https://github.com/open-runtimes/examples/tree/main/node/send_http_request) do. JSON doesn't support comments hence the RED background and white text.

Reply:
fixed